### PR TITLE
resource_config: add default values for headers

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: flask_resources/loaders.py [hunk 1]
```diff
@@ -39,12 +39,20 @@ def wrapper(self, *args, **kwargs):
...
         # Checking Content-Type is the responsibility of the loader
         loaders = self.resource.config.request_loaders
+        default_content_type = getattr(
+            self.resource.config, "default_content_type", None
+        )
+
+        request_content_type = request.content_type
+
+        if not request_content_type and default_content_type:
+            request_content_type = default_content_type
```
Comment (ppanero) [inline flask_resources/loaders.py:49]: minor: the logic is a bit messy to follow, what about the following (simpler IMHO)

```suggestion
        request_content_type = request.content_type
        
        if not request_content_type:
            request_content_type =getattr(self.resource.config, "default_content_type", None)
```

File: flask_resources/loaders.py [hunk 2]
```diff
@@ -40,11 +40,18 @@ def wrapper(self, *args, **kwargs):
...
+            request_content_type = getattr(
+                self.resource.config, "default_content_type", None
+            )
+
+        if request_content_type not in loaders:
             raise InvalidContentType(allowed_mimetypes=loaders.keys())
 
         deserializer = select_deserializer(
-            self.resource_method, loaders[request.content_type]
+            self.resource_method, loaders[request_content_type]
```
Comment (lnielsen) [inline flask_resources/loaders.py:54]: What happens if a content-type doesn't exists?

Comment (zzacharo) [reply flask_resources/loaders.py:52]: It raises an `InvalidContentType` in this [line](https://github.com/inveniosoftware/flask-resources/pull/71/files#diff-de2cf242e5126989fce8235c111e296eR49)

File: tests/conftest.py [hunk 3]
```diff
@@ -23,6 +23,8 @@ class CustomResourceConfig(ResourceConfig):
 
     item_route = "/custom/<id>"
     list_route = "/custom/"
+    default_content_type = "application/json"
+    default_accept_mimetype = "application/json"
```
Comment (fenekku) [inline tests/conftest.py:27]: We get these by inheritance, so we don't need to add them :fire:

**Pull request merged**